js/index.js

```diff
@@ -1,42 +1,92 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
+  console.log("Calculating subtotal, yey!")
```

Comment:
After you're done and submit, I recommend removing any code/comments which doesn't serve a purpose. It'll make your code minimal and easier to read